app/helpers/students_helper.rb

```diff
@@ -0,0 +1,5 @@
+module StudentsHelper
+	def res
+		redirect_to new_student_session_path
```

Comment:
We should redirect from the helper. Please use controller to do so. The main work of redirection is of the controller.